src/Oceananigans.jl

```diff
@@ -162,9 +165,65 @@ const defaults = Defaults()
     AbstractModel
 
 Abstract supertype for models.
+
+Models are required to have the following fields:
+  - `model.clock::Clock`
+  - `model.architecture`
+  - `model.timestepper` with `timestepper.G⁻` and `timestepper.Gⁿ`
```

Comment:
I don't think we want this: the `OceanSeaIceModel` does not have a timestepper

Reply:
I don't think we need it. It is used only in the `reset!(::AbstractModel)` method, which should know wether we have a timestepper or not.

Reply:
oh, actually it is currently used in the `initialize!(simulation)` step... But this should call `initialize!(model)` and not `reset!(timestepper(model))`